Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM node:14 AS development
```

Comment:
I believe this might be the base stage and then fellow stages could be start from it

Reply:
Done, currently this is the base image to the build stage